files/when/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "when",
+  "repo": "cujojs/when",
+  "files": {
+    "include": ["when.js"]

```

Comment:
That file won't work as it's importing a bunch of stuff https://github.com/cujojs/when/blob/master/when.js